README.md

```diff
@@ -75,6 +75,8 @@ you will be prompted to add your folder name and/or
 choose template option from list. For foldername, you
 can choose to use the default foldername provided in
 the prompt or type in your preferred folder name.
+Replace `<folder_name>` with your preferred folder name.
```

Comment:
You have added it to the wrong place. Also don't format the folder_name and template. Just leave everything as plain text.

Reply:
Should be fine, now:
* Add it to the right place
* Removed formatting
* Made changes to both README.md files
* added escape character before `<template>` because it just disappears without it

Reply:
@Sync271 Really? The template disappears without it? Interesting. Thank you for that 👍🏽💯 I've learned something new from your PR.
I've seen your changes. Nice. Although the issue is that you added the new text after the text you met there. The task description says it should be before, not after. There is a reason why it has to be before and not after. Can you help change this in both? files?

Reply:
I was surprised to find it out too, google a bit but found nothing on it. Weird stuff.

I've somehow missed a lot of key details before making a PR, sorry for that!
Everything should hopefully be fine now!